source/opt/common_uniform_elim_pass.h

```diff
@@ -66,6 +67,14 @@ class CommonUniformElimPass : public Pass {
   // Return true if variable is uniform
   bool IsUniformVar(uint32_t varId);
 
+  // Given an OpAccessChain instruction, return true
```

Comment:
You should say more about the assumptions:
- the base pointer is an OpVariable
- the variable is of type pointer-to-struct

Also say it only sees volatile on a member of that struct type.  (See discussion above)